packages/debug/bin/download-adapters.js

```diff
@@ -19,10 +19,31 @@ const request = require('request');
 const unzip = require('unzip-stream');
 const path = require('path');
 const process = require('process');
+const zlib = require('zlib');
```

Comment:
it won't work outside of Theia repo, dev dependencies won't be installed for clients

Reply:
Fixed